examples/Preprocessing_example.md

```diff
@@ -0,0 +1,77 @@
+# Preprocessign an Image
+
+The main objective of the Preprocessing phase is to make as easy as possible for the OCR system to distinguish a character/word from the background. There are several techniques used for Preprocessing but this **Preprocessor** class can do it without you going into details and jargon of traditional methods.
+
+`!pip install -U ocred --no-cache --force-reinstall`
+
+```
+# manually preprocessing an image
+import cv2
+from scipy import ndimage
+from ocred import Preprocessor
+from google.colab.patches import cv2_imshow
```

Comment:
I think we can use `cv2` or `matplotlib` to show images in `Jupyter`. (`cv2.imshow` should work and this import can be removed)

Reply:
I did use `cv2.imshow` initially but it gave an error that this function is not functional anymore in colab, hence I had to import the new function separately and use it.